src/Logary/Internals/InternalLogger.fs

```diff
@@ -1,14 +1,13 @@
 namespace Logary.Internals
 
-open System
 open Hopac
 open Hopac.Infixes
 open Logary
 open Logary.Internals
 open Hopac.Extensions
 
 [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
-module InternalLogger =
+module internal InternalLogger =
```

Comment:
Yeah, good. We could also consider internalising more in `Logary.Formatting` as it's a bit hairy to navigate for new users.